Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,216 @@
+                        personCount++;
+                    }
+                }
+                else if (amount > account.CurrentBalance()) // this is now the poorest account.
+                {
+                    poorestName.Clear();
+                    poorestName.Append(account.GetName());
+                    amount = account.CurrentBalance();
+                    personCount = 1;
+                }
+            }
         }
 
         public static string FindRichestPerson(string[] peopleAndBalances)
         {
-            return "";
-        }
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            StringBuilder richestName = new StringBuilder();
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
 
+                if (i == 0)
+                {
+                    richestName.Append(account.GetName());
+                    amount = account.CurrentBalance();
+                }
+                else if (amount == account.CurrentBalance())
+                {
+                    if (personCount == 1)
+                    {
+                        richestName.Append(" and " + account.GetName());
+                        personCount++;
+                    }
+                    else
+                    {
+                        richestName.Replace(" and ", ", ");
+                        richestName.Append(" and " + account.GetName());
+                        personCount++;
+                    }
+                }
+                else if (amount < account.CurrentBalance())
+                {
+                    richestName.Clear();
+                    richestName.Append(account.GetName());
+                    amount = account.CurrentBalance();
+                    personCount = 1;
+                }
+            }
+
+            return MakeStuffWithInput.MakefullstringRichestPeople(personCount, richestName.ToString(), amount);
+        }
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            StringBuilder biggestLossName = new StringBuilder();
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
+
+                if (account.MoreThan1Balance())
+                {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)